categorize_adjectives/bert_similarity_calc.py

```diff
@@ -0,0 +1,60 @@
+"""Generate dictionary of distances between each pair of adjective"""
+import os
+import pickle
+from libraries.bert_embeddings import BertEmbedding
+from scipy.spatial.distance import cosine
+
+def get_most_frequent_adjs(noun_to_adj, num_adjs):
+    """Generate the list of most frequent num_adjs in noun_to_adj dictionary"""
+    selected_adj = []
+    adj_count = {}
+    for noun in noun_to_adj.keys():
+        for adj, _ in noun_to_adj[noun]:
+            try:
+                adj_count[adj] += -1
+            except KeyError:
+                adj_count[adj] = -1
+    sorted_adjs = sorted(adj_count.items(), key=lambda kv: (kv[1], kv[0]))[:num_adjs]
+    for adj, _ in sorted_adjs:
+        selected_adj.append(adj)
+    return selected_adj
+
+def generate_distance_metric(adj_list):
```

Comment:
nit: compute_adj_similarity_matrix

Reply:
It's distance not similarity...I will change it to compute_adj_distance_matrix